src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -226,7 +226,11 @@ export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = (
   }
 
   const goBack = () => {
-    router.go(-1)
```

Comment:
The user has to press the button twice after editing the form 🤔

Reply:
but in this case, users who would add artworks from the Insights tab would be sent to my collection